docs/static/ls-to-cloud.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[connecting-to-cloud]]
+=== Sending data to Elasticsearch Service
+
+Our hosted Elasticsearch Service is available on AWS, GCP, and Azure.
+{ess-trial}[You can try the Elasticsearch Service for free].
+
+Logstash comes with two settings that simplify sending data to
+https://cloud.elastic.co/[Elastic Cloud]: Cloud ID and Cloud Auth. 
+
+[[cloud-id]]
```

Comment:
FYI, these changes appear to break the anchor links currently used in the plugin docs - they link to https://www.elastic.co/guide/en/logstash/current/connecting-to-cloud.html#_cloud_id, whereas this creates an anchor https://logstash_11884.docs-preview.app.elstc.co/guide/en/logstash/master/connecting-to-cloud.html#cloud-id, so we will need to either update this anchor, or the accompanying documentation in the existing plugin docs.

Reply:
Thanks for calling this out. This PR uses the current approach to setting up anchors. I'll open issues in the ES plugins to fix the links.  They're not broken, but they won't resolve with the precision we want after this PR is merged.